graphdatascience/session/dedicated_sessions.py

```diff
@@ -71,56 +71,26 @@ def get_or_create(
             session_id=session_id, session_connection=session_connection, db_connection=db_connection
         )
 
-    def delete(self, session_name: str, dbid: Optional[str] = None) -> bool:
-        candidate: Optional[SessionDetails] = None
-        if not dbid:
-            dbs = self._aura_api.list_instances()
-            for db in dbs:
-                candidate = self._find_existing_session(session_name, db.id)
-                if candidate:
-                    dbid = db.id
-                    break
-        else:
-            candidate = self._find_existing_session(session_name, dbid)
+    def delete(self, session_name: str) -> bool:
+        candidate = self._find_existing_session(session_name)
 
         if candidate:
-            return self._aura_api.delete_session(candidate.id, str(dbid)) is not None
+            return self._aura_api.delete_session(candidate.id) is not None
 
         return False
 
-    def list(self) -> List[SessionInfo]:
-        dbs = self._aura_api.list_instances()
-
-        sessions: List[SessionDetails] = []
-        for db in dbs:
-            try:
-                sessions.extend(self._aura_api.list_sessions(db.id))
-            except AuraApiError as e:
-                # ignore 404 errors when listing sessions
-                # it could mean paused/deleted instance or an invalid db type
-                if e.status_code == 404 or (e.status_code == 400 and "of tier" in e.message):
-                    continue
-                raise e
+    def list(self, dbid: Optional[str] = None) -> List[SessionInfo]:
+        sessions: List[SessionDetails] = self._aura_api.list_sessions(dbid)
 
         return [SessionInfo.from_session_details(i) for i in sessions]
 
-    def _find_existing_session(self, session_name: str, dbid: str) -> Optional[SessionDetails]:
+    def _find_existing_session(self, session_name: str) -> Optional[SessionDetails]:
```

Comment:
I guess we could make the dbid here optional as well. It would speed up the filter by a bit should the user have many many sessions

Reply:
would be nice. I would like to do this once we dont have AuraDS based sessions in the client anymore